lib/ed25519-browser.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as ed25519 from '@stablelib/ed25519';
+
+// browser MUST provide "crypto.getRandomValues"
+const crypto = self && (self.crypto || self.msCrypto);
+if(!crypto.getRandomValues) {
+  throw new Error('Browser does not provide "crypto.getRandomValues".');
+}
+
+// helper function used by stablelib to generate seeds
+const randomBytes = () => {
+  const seed = new Uint8Array(32);
+  return crypto.getRandomValues(seed);
+};
+
+export default {
+  ...ed25519,
+  async generateKeyPair() {
+    const seed = randomBytes();
```

Comment:
This function isn't used by stablelib, we're just using it here to generate seeds. Also, it's not a generic `randomBytes` function, rather it specifically generates 32 random bytes for a seed. That means it should be renamed `generateRandomSeed`. But I don't think we need the complexity of the function at all (I'd just put the two lines of code directly into the function below:

```suggestion
export default {
  ...ed25519,
  async generateKeyPair() {
    const seed = new Uint8Array(32);
    crypto.getRandomValues(seed);
```